conda/cli/main_export.py

```diff
@@ -114,6 +116,13 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
     if args.file is None:
         stdout_json(env.to_dict()) if args.json else print(env.to_yaml(), end="")
     else:
+        filename = args.file
+        # check for the proper file extension; otherwise when the export file is used later,
+        # the user will get a file parsing error
+        if not filename.endswith((".yml", ".yaml")):
```

Comment:
Can we use one of the yaml constants we already have defined? E.g. https://github.com/conda/conda/blob/6488061bdb0c46251bddc245d673b119e8c12e16/conda/common/configuration.py#L1310

Reply:
I wasn't aware of this constant, thanks for pointing it out!